internal/identity/gitlab.go

```diff
@@ -0,0 +1,96 @@
+package identity // import "github.com/pomerium/pomerium/internal/identity"
+
+import (
+	"context"
+	"fmt"
+	"github.com/pomerium/pomerium/internal/httputil"
+	"github.com/pomerium/pomerium/internal/sessions"
+	"net/http"
+	"net/url"
+)
+import oidc "github.com/pomerium/go-oidc"
```

Comment:
combine imports from above, and it's typical to group imports like this: https://stackoverflow.com/a/38714480